tests/test/scheduler/test_function_client_server.cpp

```diff
@@ -16,6 +16,13 @@
 
 using namespace scheduler;
 
+static void tearDown(FunctionCallClient& cli, FunctionCallServer& server)
```

Comment:
I have decided to be verbose, and add a `tearDown` function per file. Hopefuly this aids when moving to text fixtures as discussed off line.

Reply:
I think we'd have a `FunctionCallClientServerFixture` that would set up both the client and server, then perform this tear down in its destructor. This should be relatively easy to configure similar to the [`DistTestsFixture`](https://github.com/faasm/faabric/blob/master/tests/dist/fixtures.h). This would save a couple of lines of repeated setup code in the tests in this file.